Fakebook.Profile/Fakebook.Profile.Domain/DomainProfile.cs

```diff
@@ -14,8 +14,8 @@ public class DomainProfile
         //[anything]@[anything].[anything]
         private string _email;
 
-        public IList<string> FollowerEmails { get; set; }
-        public IList<string> FollowingEmails { get; set; }
+        public IList<DomainProfile> Followers { get; set; }
+        public IList<DomainProfile> Following { get; set; }
```

Comment:
the tradeoff here is, if we let profiles directly reference other profiles, then when we want to add a follow, we have to load the entire profiles of all the other follows from the database, instead of just a unique ID of theirs. i'll leave it up to the team.